dynacred/src/Storage.ts

```
@@ -32,13 +32,22 @@ export class StorageDB {
     }
 
     static async getObject(entry: string): Promise<any> {
-        const obj = JSON.parse(await this.get(entry), (key, value) => {
-            if (value && typeof value === "object" && value.type === "Buffer") {
-                return Buffer.from(value);
-            }
-            return value;
-        });
-        return obj == null ? {} : obj;
+        const d = await this.get(entry);
+        if (d === "") {
+            return {};
+        }
+        try {
+            const obj = JSON.parse(d, (key, value) => {
+                if (value && typeof value === "object" && value.type === "Buffer") {
+                    return Buffer.from(value);
+                }
+                return value;
+            });
+            return obj == null ? {} : obj;
+        } catch (e) {
+            Log.catch(e, "couldn't parse data", d);
```

Catch only `SyntaxError`, to avoid masking other errors.